assets/js/modules/analytics/datastore/setup-flow.test.js

```diff
@@ -248,46 +251,15 @@ describe( 'modules/analytics setup-flow', () => {
 				expect( registry.select( MODULES_ANALYTICS ).getSetupFlowMode() ).toBe( 'ua' );
 			} );
 
-			it( 'should return undefined if selected account returns an empty array from UA getProperties selector', () => {
+			it( 'should return undefined if selected account returns undefined from UA getProperties selector', () => {
```

Comment:
@tofumatt  This test was totally wrong!

It should be tested for a loading state (one of the new tests we added)

So the test needs to set up all the GA4 stuff (I'm explicitly asserting that `isAdminAPIWorking` is true)

And then UA will be loading.   :+1: